spot-client/src/common/remote-control/remote-control-service.js

```diff
@@ -208,6 +208,17 @@ class RemoteControlService extends EventEmitter {
             });
     }
 
+    /**
+     * Requests a Spot-TV to change its audio level to a specific direction.
+     *
+     * @param {string} direction - The direction of the volume adjustment. One of 'up' or 'down'.
+     * @returns {Promise} Resolves if the command has been acknowledged.
+     */
+    adjustVolume(direction) {
```

Comment:
Maybe this should be a number instead of a direction? Like +1, -1? That's not needed now though.

Reply:
Probably. This is a matter of world view :) +1 and -1 makes the code easier inside the volume button and the signalling, but calling them 'up' and 'down' makes it a bit more abstract and makes the external API more human friendly. Both are good approaches, I think. Let's revisit this later, and change it if need be